### PR TITLE
fix EC2 credential policies and multicloud account creation workflow

### PR DESCRIPTION
**Pull request by sirajudheenam**
- fixes cronus issue # - 417 - [cronus ui] - Users without permissions are able to access pages

### CODE REVIEW

File: config/esbuild/build.js [hunk 1]
```diff
@@ -125,7 +125,8 @@ const config = {
     },
   ],
   //loader: { ".js": "jsx" },
-  target: ["es6", "chrome58", "firefox57", "safari11", "edge18"],
+  // target: ["es6", "chrome58", "firefox57", "safari11", "edge18"],
+  target: ["es2020"],
```
Comment (ArtieReus) [inline config/esbuild/build.js:129]: why did you remove all other ones? did you align it with @andypf ?

File: yarn.lock [hunk 2]
```diff
@@ -909,6 +1344,357 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@smithy/abort-controller@^2.0.12":
```
Comment (ArtieReus) [inline yarn.lock:1347]: Are oll of those entries dependencies from aws?

File: yarn.lock [hunk 3]
```diff
@@ -15,6 +15,441 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
+"@aws-crypto/crc32@3.0.0":
```
Comment (ArtieReus) [inline yarn.lock:18]: Are all of this packages required? This is a lot....

**Review by ArtieReus — CHANGES_REQUESTED** [review]

File: config/esbuild/build.js (hunk 1 above)
Comment (edda) [reply config/esbuild/build.js:129]: Please no changes outside of your plugin without discussing it first!

File: yarn.lock (hunk 3 above)
Comment (sirajudheenam) [reply yarn.lock]: Removed them. It is supposed to be in my new plugin not here.

File: yarn.lock (hunk 2 above)
Comment (sirajudheenam) [reply yarn.lock]: Not sure. I have reverted the upstream.

File: config/esbuild/build.js (hunk 1 above)
Comment (sirajudheenam) [reply config/esbuild/build.js]: reverted back. not needed for this branch.

File: config/esbuild/build.js [hunk 4]
```diff
@@ -1,51 +1,51 @@
 #!/usr/bin/env node
-const pathsResolverPlugin = require("./paths_resolver_plugin")
-const globImportPlugin = require("./glob_import_plugin")
-const postcss = require("postcss")
-const sass = require("sass")
-const url = require("postcss-url")
-const fs = require("fs")
+const pathsResolverPlugin = require('./paths_resolver_plugin');
```
Comment (ArtieReus) [inline config/esbuild/build.js:2]: Hi sam! It looks like you are not anymore using our predefined settings for formatting the js files. Please use our predefined settings.

File: package.json [hunk 5]
```diff
@@ -99,4 +99,4 @@
   "version": "2.0.0",
   "repository": "https://github.com/sapcc/elektra.git",
   "main": "index.js"
-}
+}
```
Comment (ArtieReus) [inline package.json:102]: why there is a change on this file? Empty line?

File: yarn.lock [hunk 6]
```diff
@@ -8261,4 +8261,4 @@ yauzl@^2.10.0:
 yocto-queue@^0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
-  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```
Comment (ArtieReus) [inline yarn.lock:8264]: same same...

**Review by ArtieReus — CHANGES_REQUESTED** [review]

File: config/esbuild/build.js (hunk 4 above)
Comment (sirajudheenam) [reply config/esbuild/build.js:2]: Reverted the file. Please have a look. https://github.com/sirajudheenam/elektra/blob/ec2_credential_fix_26_oct_23/config/esbuild/build.js

File: package.json (hunk 5 above)
Comment (sirajudheenam) [reply package.json:102]: Change is reverted already. https://github.com/sirajudheenam/elektra/blob/ec2_credential_fix_26_oct_23/package.json#L102

Comment (ArtieReus) [inline package.json:102]: I also see changes here. Can you remove them?

File: yarn.lock (hunk 6 above)
Comment (ArtieReus) [inline yarn.lock:8264]: I also see changes here although the line seems to be equal

**Review by ArtieReus — CHANGES_REQUESTED** [review]

**Review by ArtieReus — CHANGES_REQUESTED** [review]
